src/ray/raylet/lib/python/raylet_extension.cc

```diff
@@ -367,83 +358,82 @@ static PyObject *PyLocalSchedulerClient_free(PyObject *self, PyObject *args) {
     object_ids.push_back(object_id);
   }
 
-  // Invoke local_scheduler_free_objects_in_object_store.
-  local_scheduler_free_objects_in_object_store(
-      reinterpret_cast<PyLocalSchedulerClient *>(self)->local_scheduler_connection,
-      object_ids, local_only);
+  auto client = reinterpret_cast<PyRayletClient *>(self)->raylet_client;
+  // Invoke raylet_FreeObjects.
+  RAY_CHECK_OK(client->FreeObjects(object_ids, local_only));
   Py_RETURN_NONE;
 }
 
-static PyMethodDef PyLocalSchedulerClient_methods[] = {
-    {"disconnect", (PyCFunction)PyLocalSchedulerClient_disconnect, METH_NOARGS,
+static PyMethodDef PyRayletClient_methods[] = {
+    {"Disconnect", (PyCFunction)PyRayletClient_Disconnect, METH_NOARGS,
```

Comment:
@suquark @pcmoritz the Python function names should still be formatted like `submit_task` (lower case with underscores) I think.

Reply:
ok, I have restored their names.